src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private String[] colors = {"red", "blue", "white", "black", "pink", "green"};
+
+    public String getRandomColor() {
+        return colors[(int)(Math.random() * 5)];
```

Comment:
```suggestion
        return colors[(int)(Math.random() * colors.length)];
```
this way if you add/remove smt from the colors it won't break this code

Reply:
super!